# Population state message
# This structure contains the state of the object population.

# What is the current state of the plugin ("running", "paused")?
bool enabled
